<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentInfo</class>
 <widget class="QDialog" name="StudentInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelUsername">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMyData">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Hae omat tiedot</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMyGrades">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>140</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Arvosanat</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
